lib/dialects/mssql/index.js

```diff
@@ -31,7 +31,6 @@ function Client_MSSQL(config = {}) {
     min: 1,
     max: 1,
     idleTimeoutMillis: Number.MAX_SAFE_INTEGER,
-    evictionRunIntervalMillis: 0,
```

Comment:
@elhigu @koskimas I checked tarn.js and it appear that replicating behaviour of the old pool is no longer possible here, setting `reapIntervalMillis` to 0 is not supported. Was that intentional?

Reply:
Related: https://github.com/Vincit/tarn.js/issues/56

Reply:
That option is not needad because min: 1, max: 1 already prevents idle connection from being discarded anyways.